### DISCUSSION TITLE
Unable to delete custom domain completely

### DISCUSSION THREAD

**Discussion by Pathsis** [vercel/community#136] [Help] [answered]
Hi, I deleted my project deployed on Vercel (imported from github Pages repository) and removed all custom domain settings, but I still **_intermittently_** received the following message when I accessed the domain I had unconnected to Vercel on my phone’s broswer:

```
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: fra1::5wklj-1639643133276-60184dd1880f
```

Should I be able to completely remove my domain from Vercel?

_I have cleared all my cookies cache. I deleted all the relevant Settings on my DNS server._

**Comment by den-sokolovus** [comment, marked as answer]
Hey Pathsis, thanks for the question! The info about the IP address the custom domain was pointed to (A record) could be cached in your phone's browser. Thus, when trying to access it, you're routed to the cached IP, and get the error since the project was deleted. Did you try to clear the cache of your phone's browser?
It would be also great to learn the domain name if you find this appropriate. Thank you!

**Comment by Pieparker** [comment]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
